src/components/CustomButton/styles.ts

```diff
@@ -3,21 +3,21 @@ import {colors, dimensions, text} from '../../constants';
 
 export const styles = StyleSheet.create({
   button: {
-    minWidth: '60%',
+    width: '100%',
```

Comment:
for futures - we are going to enforce using horizontalScale, verticalScale, moderateScale on all of our mobile projects. But it's fine to use 100% if you want to expand button to the full width. (maybe flex: 1 would also work)